app/views/prison/visits/_prisoner_details.html.erb

```diff
@@ -64,13 +64,36 @@
   </div>
 </div>
 
-<% if @visit.prisoner_restrictions.any? %>
-  <div class="border-box border-box--large border-box--orange push-top">
-    <h3 class="bold-medium">
-      <%= t('.prisoner_restrictions_apply') %><div class="push-left--half font-xsmall tag tag--verified"><%= t('.verified') %></div>
-    </h3>
-    <% @visit.prisoner_restrictions.each do |restriction| %>
-      <%= render 'prisoner_restriction', restriction: restriction %>
+
+<div class="grid-row push-top">
+  <div class="column-one-quarter">
+    <% if @visit.prisoner_restrictions.any? %>
+      <div class="border-box border-box--large border-box--orange">
+        <h3 class="bold-medium">
+          <%= t('.prisoner_restrictions_apply') %><div class="push-left--half font-xsmall tag tag--verified"><%= t('.verified') %></div>
+        </h3>
+        <% @visit.prisoner_restrictions.each do |restriction| %>
+          <%= render 'prisoner_restriction', restriction: restriction %>
+        <% end %>
+      </div>
+    <% else %>
+      &nbsp;
```

Comment:
@chrishanes is this an exceptable hack for you?

Reply:
Why are the restrictions in a column 1/4 now?

Reply:
@chrishanes

![o-no-idea-dog-meme-facebook](https://user-images.githubusercontent.com/100796/33178707-fd576bd6-d05e-11e7-821f-7b846f8117dd.jpg)

Reply:
doing